client/apps/edit/components/layout/index.coffee

```diff
@@ -116,7 +116,7 @@ module.exports = class EditLayout extends Backbone.View
 
   getBodyText: =>
     @fullText = []
-    if @article.get('lead_paragraph').length
+    if @article.get('lead_paragraph')?.length
```

Comment:
Agree 100%! I definitely feel the `allow(null,"")` pain these days and it'd be great to address this as technical debt after/while we're doing this article overhaul. I'll make an issue in the meantime.